internal/utils/persistence/src/main/java/org/eclipse/ditto/internal/utils/persistence/mongo/config/MongoDbConfig.java

```diff
@@ -203,6 +210,11 @@ enum OptionsConfigValue implements KnownConfigValue {
              */
             RETRY_WRITES("retryWrites", true),
 
+            /**
+             * Determines whether IAM role should be used for authentication.
+             */
+            USE_AWS_IAM_ROLE("useAwsIamRole", false),
```

Comment:
Please also add this as configuration key (same as the above `retryWrites`) to this file, adding a one-line documentation in the config as well:
https://github.com/eclipse-ditto/ditto/blob/d5f43d0c7262154ae724b39d42ed9582b862f33b/internal/utils/config/src/main/resources/ditto-mongo.conf#L50-L52

This is then also the place to define the environment variable `USE_AWS_IAM_ROLE` which you already use in the Helm chart:
```hocon
useAwsIamRole = false
useAwsIamRole = ${?USE_AWS_IAM_ROLE}
```